apps/forge/forge.app.js

```diff
@@ -0,0 +1,32 @@
+// App Forge
+
+st = require('Storage');
+
+l = /^a\..*\.js$/;
+//l = /.*\.js/;
+l = st.list(l, {sf:false});
+
+print(l);
+
+function on_load(x) {
+  print("Loading", x);
+  Bangle.buzz(50, 1); // Won't happen because load() is quicker
+  g.reset().clear()
+    .setFont("Vector", 40)
+    .drawString("Loading", 0, 30)
+    .drawString(x, 0, 80);
+  g.flip();
+  load(x);
+}
+
+var menu = {
+    "< Back" : Bangle.load
+};
+if (Object.keys(l).length==0) Object.assign(menu, {"No apps":""});
```

Comment:
If you like, this can be shortened:

```suggestion
if (l.length==0) Object.assign(menu, {"No apps":""});
```